src/zombiereborn.cpp

```diff
@@ -117,145 +301,190 @@ void CZRPlayerClassManager::LoadPlayerClass()
 	m_vecZombieDefaultClass.Purge();
 	m_vecHumanDefaultClass.Purge();
 
+	// attempt to load KV1 config for backwards compatibility
 	KeyValues* pKV = new KeyValues("PlayerClass");
 	KeyValues::AutoDelete autoDelete(pKV);
 
-	const char *pszPath = "addons/cs2fixes/configs/zr/playerclass.cfg";
+	const char *pszKV1Path = "addons/cs2fixes/configs/zr/playerclass.cfg";
+	bool bUsingKV1Config = true;
 
-	if (!pKV->LoadFromFile(g_pFullFileSystem, pszPath))
+	if (!pKV->LoadFromFile(g_pFullFileSystem, pszKV1Path))
 	{
-		Warning("Failed to load %s\n", pszPath);
-		return;
+		Message("Failed to load %s, attempting to load json playerclass config\n", pszKV1Path);
+		bUsingKV1Config = false;
 	}
 
-	for (KeyValues* pKey = pKV->GetFirstSubKey(); pKey; pKey = pKey->GetNextKey())
+	ordered_json jPlayerClasses;
+
+	if (bUsingKV1Config)
+		jPlayerClasses = KV1PlayerClassConfigToJson(pKV);
+	else
 	{
-		bool bHuman = !V_strcmp(pKey->GetName(), "Human");
+		const char *pszJsonPath = "addons/cs2fixes/configs/zr/playerclass.json";
```

Comment:
I think you may have misunderstood a little when I explained the backwards compatibility, we really only want to be supporting one primary config format as it's unneeded long-term complexity otherwise. Rather, the JSON converter should automatically save a new `playerclass.json` converted from an old `playerclass.cfg` if it exists, and then delete the old file (may as well delete `playerclass.cfg.example` if it's there too, to avoid future confusion).

This way we can remove the conversion code after a couple releases/months, once all servers have had a chance to run the newer versions at least once. Remove `playerclass.cfg.example` from the repo as well.